adafruit_is31fl3731.py

```diff
@@ -115,8 +169,17 @@ def autoplay(self, delay=0, loops=0, frames=0):
         self._register(_CONFIG_BANK, _AUTOPLAY1_REGISTER, loops << 4 | frames)
         self._register(_CONFIG_BANK, _AUTOPLAY2_REGISTER, delay % 64)
         self._mode(_AUTOPLAY_MODE | self._frame)
+        return
+
```

Comment:
This not is not needed either, since the return is empty.

Reply:
ok